### PR TITLE
Several fixes

### PR DESCRIPTION
**Pull request by fernandog**
several codacy issues fix

### CODE REVIEW

File: sickbeard/providers/torrent/json/torrentday.py [hunk 1]
```diff
@@ -185,29 +185,33 @@ def login(self):
             logger.log('Failed to login, you must add your cookies in the provider settings', logger.WARNING)
             return False
 
-            login_params = {
```
Comment (fernandog) [inline sickbeard/providers/torrent/json/torrentday.py]: fixes "Unreachable code"

https://www.codacy.com/app/fernandog/SickRage/file/3942279571/issues/source?bid=3716179&fileBranchId=3716179#l188

File: sickbeard/server/web/core/base.py [hunk 2]
```diff
@@ -135,7 +135,7 @@ def render(self, *args, **kwargs):
             kwargs['title'] = '500'
             kwargs['header'] = 'Mako Error'
             kwargs['backtrace'] = RichTraceback()
-            for (filename, lineno, function, line) in kwargs['backtrace'].traceback:
+            for (filename, lineno, function, _) in kwargs['backtrace'].traceback:
```
Comment (fernandog) [inline sickbeard/server/web/core/base.py]: is this ok?

File: sickbeard/__init__.py [hunk 3]
```diff
@@ -1235,6 +1233,7 @@ def initialize(consoleLogging=True):  # pylint: disable=too-many-locals, too-man
         POSTER_SORTBY = check_setting_str(CFG, 'GUI', 'poster_sortby', 'name')
         POSTER_SORTDIR = check_setting_int(CFG, 'GUI', 'poster_sortdir', 1)
         DISPLAY_ALL_SEASONS = bool(check_setting_int(CFG, 'General', 'display_all_seasons', 1))
+        RECENTLY_DELETED = set()
```
Comment (fernandog) [inline sickbeard/__init__.py]: @ratoaq2 issue still exists. any idea?

https://www.codacy.com/app/fernandog/SickRage/file/3945997134/issues/list?bid=3716968&fileBranchId=3716968

File: sickbeard/naming.py [hunk 4]
```diff
@@ -65,10 +65,7 @@ def _is_anime(self):
         Find out if show is anime
         :return: True if show is anime, False if not
         """
-        if self.anime > 0:
-            return True
-        else:
-            return False
+        return bool(self.anime)
```
Comment (ratoaq2) [inline sickbeard/naming.py]: what if self.anime was `-1` before?

The old code returns True, now it returns False.

Is it possible? Was it a bug before?

File: sickbeard/notifiers/kodi.py [hunk 5]
```diff
@@ -602,7 +602,4 @@ def update_library(self, showName=None):
 
             # needed for the 'update kodi' submenu command
             # as it only cares of the final result vs the individual ones
-            if result == 0:
-                return True
-            else:
-                return False
+            return bool(result)
```
Comment (ratoaq2) [inline sickbeard/notifiers/kodi.py]: This is a bug.
Before it was only returning True if result is zero. Now it's the other way around.

Comment (fernandog) [inline sickbeard/notifiers/kodi.py]: fixed in latest commit

File: sickbeard/naming.py (hunk 4 above)
Comment (fernandog) [inline sickbeard/naming.py]: i think its a bug. The column value is 0 or 1

**Comment by fernandog** [comment]
@ratoaq2 are we ok to merge this into develop

**Comment by fernandog** [comment]
Yes.

**Pull request merged**